other/median_of_two_arrays.py

```diff
@@ -1,18 +1,17 @@
 # This code finds the median of two arrays, even if they are not sorted initially
 def find_median_arrays(nums1, nums2):
```

Comment:
OK...  There is a lot here so let me explain...
1. We add [type hints](https://docs.python.org/3/library/typing.html) to say that `nums` are list of ints and the the function returns an int.
2. We add [doctests](https://docs.python.org/3/library/doctest.html) which we can run locally with the command `python3 -m doctest -v median_of_two_arrays.py`
```suggestion
from typing import List


def median_of_two_arrays(nums1: List[int], nums2: List[int]) -> int:
    """
    >>> median_of_two_arrays((9, 1), (7, 3, 5))
    5
    >>> median_of_two_arrays([1], [0, 1000])
    1
    >>> median_of_two_arrays([-10], [10])
    0
    >>> median_of_two_arrays([-10], [10, 5])
    5
    >>> median_of_two_arrays([0], [3])
    1
    >>> median_of_two_arrays([], [0])
    0
    >>> median_of_two_arrays([], [])
    Traceback (most recent call last):
      ...
    IndexError: list index out of range
    """
```
Are the answers in the doctests what we expect them to be?